### PR TITLE
add new /DT/BRICK/DEL/1 to delete solid elements by geometry criteria

### PR DESCRIPTION
**Pull request by qiangRadioss**
#### Description of the changes

add new engine input /DT/BRICK/DEL/1 to delete solid&thick shell elements by geometry criteria

### CODE REVIEW

File: engine/share/modules/dt_mod.F [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            PROCEDURE, PASS :: INIT_IDEL_BRICK         
+        END TYPE DT_
+        
+C-----------------------------------------------        
+		TYPE(DT_) :: DT
+C-----------------------------------------------                
+        
+        CONTAINS        
+          
+		  !INITIALISATION : CALL DT%INIT_IDEL_BRICK()
```
Comment (sebastienVilleneuve) [inline engine/share/modules/dt_mod.F:22]: @qiangRadioss 

some tabulation to remove

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]

File: engine/share/modules/dt_mod.F [hunk 2]
```diff
@@ -0,0 +1,36 @@
+      MODULE DT_MOD
```
Comment (sebastienVilleneuve) [inline engine/share/modules/dt_mod.F:1]: need to discuss with @OlivierWienholtz  , not fortran 90 in this part

Comment (qiangRadioss) [reply engine/share/modules/dt_mod.F:1]: waiting for the modification on dt_mod.F, maybe Olivier do it directly?

thanks.

**Comment by sebastienVilleneuve** [comment]
@qiangRadioss 

still 5 tests are ko 1.57 1.59 1.78 1.79 1.86 .

Sébastien

**Comment by sebastienVilleneuve** [comment]
@qiangRadioss 

still 5 tests are ko 1.57 1.59 1.78 1.79 1.86 .

Sébastien

**Comment by sebastienVilleneuve** [comment]
@qiangRadioss 

still 5 tests are ko 1.57 1.59 1.78 1.79 1.86 .

Sébastien

**Comment by sebastienVilleneuve** [comment]
@qiangRadioss 

still 5 tests are ko 1.57 1.59 1.78 1.79 1.86 .

Sébastien

**Review by sebastienVilleneuve — APPROVED** [review]

**Pull request merged**